docs/manual/programmability/lua-api.md

```diff
@@ -479,6 +479,15 @@ You can use the following flags and instruct the server to treat the scripts' ex
     Redis allows scripts to be executed both in standalone and cluster modes.
     Setting this flag prevents executing the script against nodes in the cluster.
 
+* `allow-cross-slot-keys`: The flag that allows a script to access keys from multiple slots.
```

Comment:
maybe add a bullet to the list at the top?
next to `They can run in cluster mode`.

also, maybe we wanna mention it in https://redis.io/docs/manual/programmability/eval-intro/#eval-flags
i.e. state that EVAL script that didn't declare shebang has different behavior.